app/views/course/experience_points_records/index.json.jbuilder

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+json.rowCount @experience_points_count
+json.name @course_user.name
+
+json.rowData @experience_points_records do |experience_points_record|
+  json.id experience_points_record.id
+  updater_course_user = @course_user_preload_service.course_user_for(experience_points_record.updater)
```

Comment:
Should be `@course_user_preload_service.course_user_for(experience_points_record.updater) || experience_points_record.updater` in case the course user does not exist anymore. So if it is not a course user, you should render the user link instead in the frontend